bokeh/server/configure.py

```diff
@@ -88,7 +88,7 @@ def configure_flask(config_argparse=None, config_file=None, config_dict=None):
 def register_blueprint():
     global registered
     if registered:
-        warnings.warn(
+        logging.warn(
```

Comment:
AFAIK (and I just verified) the default log level is "warn", so if the user does not set the log level, all warnings are shown. If the user sets the log level to 'error' or higher, the message to `logger.warn()` is not shown, but I think we can assume that this is what the user wants then?